core/modules/field_ui/tests/field_ui.test

```diff
@@ -846,6 +846,67 @@ class FieldUIAlterTestCase extends BackdropWebTestCase {
   }
 }
 
+/**
+ * Tests for correct messaging, and DB cleanup when field creation fails.
+ *
+ * @see based on FieldUIManageFields class.
+ */
+class FieldUIManageFieldsFailTestCase extends BackdropWebTestCase {
+  function setUp() {
+    parent::setUp();
+
+    $admin_user = $this->backdropCreateUser(array('access content', 'administer content types', 'administer fields', 'administer users'));
```

Comment:
It turned out that the admin user needs only two permissions. As this still would lead to a line wider than 80 chars:

```
    $admin_user = $this->backdropCreateUser(array(
      'administer content types',
      'administer fields',
    ));
```
(That's what "each element should be broken into its own line" means, but I actually think you know that anyway. :wink: )